docs/reference/api/drivers/LowPowerTimeout.md

```diff
@@ -2,12 +2,19 @@
 
 <span class="images">![](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/classmbed_1_1_low_power_timeout.png)<span>LowPowerTimeout class hierarchy</span></span>
 
-[Add description here.]
+Use the LowPowerTimeout interface to set up an interrupt to call a function after a specified delay. You can create any number of LowPowerTimeout objects. This allows multiple outstanding interrupts at the same time.
```

Comment:
Query: When would a user want to create LowPowerTimeout objects? When would a user want to call a function after a specified delay? What's the use case here?